tests/output/pjs/test_utils.js

```diff
@@ -122,9 +122,11 @@ var runTest = function(options) {
                     checkAssertions(options.assertions2,
                         output.results.assertions);
                     options.simulateClick && simulateClick();
+                    output.output.kill();
                     done();
                 });
             } else {
+                output.output.kill();
```

Comment:
Cleanup before running the next test.  kill() now calls noLoop() which clears the interval that p5.js uses for ```draw```.  This resulted in a significant reduction in time to run the tests.  They used to take about a 90 seconds for me, now they take about 45 seconds.